ci-operator/step-registry/hypershift/mce/agent/conformance/hypershift-mce-agent-conformance-workflow.yaml

```diff
@@ -0,0 +1,37 @@
+workflow:
+  as: hypershift-mce-agent-conformance
+  documentation: |-
+    The HyperShift agent conformance workflow executes tests against a new ephemeral
+  steps:
+    allow_best_effort_post_steps: true
+    allow_skip_on_success: true
+    post:
+    - chain: hypershift-mce-agent-destroy
+    - ref: baremetalds-packet-gather-metadata
+    - chain: baremetalds-ipi-post
+    test:
+    - chain: hypershift-conformance
+    pre:
+    - ref: assisted-baremetal-conf-devscripts
+    - ref: assisted-baremetal-conf-user-data
+    - chain: baremetalds-ipi-pre
+    - ref: enable-qe-catalogsource
+    - chain: hypershift-mce-agent-lvm
+    - ref: hypershift-mce-install
+    - chain: hypershift-mce-agent-create
+    env:
+      LVM_OPERATOR_SUB_SOURCE: qe-app-registry
+      PACKET_PLAN: "m3.large.x86"
+      USER_DATA_FILENAME: "user-data"
+      NUM_EXTRA_WORKERS: "3"
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
```

Comment:
Only to remark that this covers IPv4. IPv6 and Dual stack will be covered in the nexts PRs (no action needed)